### ISSUE TITLE
sulu:admin:download-build fails on windows

### ISSUE THREAD

**Issue by lightglitch** [sulu/sulu#4901]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 2.0.2

#### Actual Behavior
```
In ResponseTrait.php line 235:
                                                                                                                              
  HTTP/1.1 400 Bad Request returned for "https://raw.githubusercontent.com/sulu/skeleton/2.0.2%5Cassets%5Cadmin%5Cindex.js".  
                                                                                                                              
```

#### Expected Behavior

Should download the admin build files.

#### Possible Solutions

The problem is that the DIRECTORY_SEPARATOR is being used to build urls in the getRemoteFileHash function and in windows the DIRECTORY_SEPARATOR  is "\\" not "/".
A solution would be:
```
$path = str_replace(DIRECTORY_SEPARATOR, "/", $path);
```
in that function.

**Comment by alexander-schranz** [comment]
@lightglitch  thank you for your report . You are correct here should not the DIRECTORY_SEPERATOR be used.

**Comment by danrot** [comment]
@lightglitch Would you mind creating a PR with your suggested change? Then we can be sure that it is actually working, since you seem to have a windows machine, and I do not :see_no_evil:

**Comment by danrot** [comment]
Fixed in #4910
